roles/services/tasks/main.yml

```diff
@@ -10,6 +10,13 @@
   with_items:
     - postfix
 
+- name: Add default SYS_CHROOT capabilities to /usr/share/containers/containers.conf file
+  lineinfile:
+    path: /usr/share/containers/containers.conf
+    regexp: '^default_capabilities = \['
+    line: "default_capabilities = [\n  \"SYS_CHROOT\","
```

Comment:
I'm not an Ansible expert, but what about the ending ']'? Or does it depend on '^default_capabilities' to exist (un-commented already)?

Reply:
Copying our file to /etc/containers/containers.conf updates only relevant section and specific item